ui/ecsconfig.py

```diff
@@ -480,25 +480,33 @@ def add_all():
         return None
 
     if l:
```

Comment:
👍 And yeah, pretty much anything you don't like or is broken, feel free to call it out in an issue.  IMO, issues are how we keep improving.